### ISSUE TITLE
Swagger UI doesn't work on public page

### ISSUE THREAD

**Issue by kitzbergerg** [ElektraInitiative/PermaplanT#314]
## Steps to Reproduce the Problem

Visit <https://www.permaplant.net/api/doc/swagger/ui>.

## Expected Result

When running the backend locally you should see the swagger-ui there (uses the port of the backend).
![image](https://user-images.githubusercontent.com/33102273/234205375-0074c30d-139c-425f-89a0-c0477018df24.png)

## Actual Result

An empty page of PermaplanT.
![image](https://user-images.githubusercontent.com/33102273/234205163-8d868dec-db93-41c2-ba2c-b41dcc8c75bb.png)

**Comment by lukashartl** [comment]
in #191 it was `/doc/api`.

So now it is `/api/doc`?

Just to be sure: Can you tell me how the URL is if you run it locally?

**Comment by lukashartl** [comment]
@kitzbergerg

For me this works:
* https://www.permaplant.net/doc/api/swagger/ui/ (the trailing slash is very important for the backend)

**Comment by markus2330** [comment]
For me this works, too.

We should put the link to our documentation (probably main README?). I'll take this over.

**Comment by kitzbergerg** [comment]
Ah yes it now works. Seems to have been a caching issue in my browser. I had to actually clear my application cache.

@markus2330 I would put the doc URL in the main README as well (like so: https://github.com/actix/actix-web#documentation. It always nice to have a quick overview at the top). 
I also made a short description in this PR: https://github.com/ElektraInitiative/PermaplanT/pull/307/files#diff-6c9a0a3a297d85aeae6b201e8aac0b5331e659a80cd46c7384271e86f67513e9R9. You might want to add it there as well (although we could also say this is just the technical description of how to build it.
